api/run.py

```diff
@@ -638,23 +638,26 @@ def saveImg():
     if file:
         filename = secure_filename(file.filename)
         file.save(filename)
-        with open(filename , "rb") as byteFile:
+        with open(filename, "rb") as byteFile:
             f = byteFile.read()
             fileId = fs.put(f)
-            print(fileId)
-            studentsDOM.addNewFile(studentId, fileId,filename)
-            studentId = ObjectId(request.args.get('studentId'))
+            studentsDOM.addNewFile(studentId, fileId, filename)
             files = studentsDOM.getFiles(studentId)
 
             cleanFiles=[]
 
-            for file in files:
```

Comment:
@jchin01 this `for file in files` did not depend on the `byteFile` from `with open` so moved it out